awx/ui/client/src/templates/job_templates/multi-credential/multi-credential-modal.directive.js

```diff
@@ -111,7 +111,7 @@ function multiCredentialModalController(GetBasePath, qs, MultiCredentialService)
 
         scope.credentialTypes.forEach((credentialType => {
             if(credentialType.kind
-                .match(/^(machine|cloud|net|ssh|vault)$/)) {
+                .match(/^(machine|cloud|net|ssh|vault|kubernetes)$/)) {
```

Comment:
Icon is missing for prompts
![Screenshot from 2020-07-16 09-21-30](https://user-images.githubusercontent.com/9753817/87676870-1f4d8180-c747-11ea-88f1-b93bd8247a51.png)

That's rendered [here](https://github.com/ansible/awx/blob/devel/awx/ui/client/src/templates/prompt/steps/credential/prompt-credential.partial.html#L14) using `promptData.prompts.credentials`, which I _think_ ultimately comes from [here](https://github.com/ansible/awx/blob/devel/awx/ui/client/lib/components/relaunchButton/relaunchButton.component.js#L87)